Add note regarding association alias casing in contain() calls. (#6691)
cakephp/docs: en/orm/retrieving-data-and-resultsets.rst

    $query = $articles->find();
    $query->contain(['Authors', 'Comments'], true);
    
.. note::

>     You are supposed to use the association alias with proper casing in ``contain()`` calls,
>     not the property named used to hold the associated record(s). So for e.g. if you have
>     declared an assocation as ``belongsTo('Users')`` then you must use ``contain('Users')``
>     and not ``contain('users')`` or ``contain('user')``.

Reviewer: ```suggestion
    Association names in ``contain()`` calls should use the same association casing as
    in your association definitions,  not the property name used to hold the association record(s).
    For example, if you have declared an assocation as ``belongsTo('Users')`` then you must 
    use ``contain('Users')`` and not ``contain('users')`` or ``contain('user')``.
```
